libbeat/outputs/kafka/config.go

```diff
@@ -14,8 +14,6 @@ type kafkaConfig struct {
 	TLS             *outputs.TLSConfig `config:"tls"`
 	Timeout         time.Duration      `config:"timeout"             validate:"min=1"`
 	Worker          int                `config:"worker"              validate:"min=1"`
-	UseType         bool               `config:"use_type"`
-	Topic           string             `config:"topic"`
```

Comment:
Yes, Topic still exists, but not required here. `outil.BuildSelectorFrom...` looks for `topic` and `topics` building a topic-selector similar to pipelining/index selection in elasticsearch output.